docs/docusaurus/versioned_docs/version-1.5.0/howtos/inbound_roaming.md

```diff
@@ -17,6 +17,8 @@ other operators to have direct connectivity to their HSS through `S6a`
 (diameter) and PGW through `S8` (gtp) interfaces. VPN is suggested to reach
 those roaming services but is out of scope of Magma.
 
+At the bottom of this document you have sample configuration
```

Comment:
Might be useful to try to link to the section below
```
Skip to the [Configuration Example](#configuration-example) for a sample configuration
```
maybe?

Reply:
yes, good catch!